From fa236a9373136576b531225923d4f2aaafa9a3ae Mon Sep 17 00:00:00 2001
From: Vikas Gupta <vikasg@netlogicmicro.com>
Date: Fri, 26 Aug 2011 17:39:07 +0530
Subject: [PATCH 245/565] MMC/SD: work with new IRT

MMC/SD: Modify to work with new IRT.

Based on Broadcom SDK 2.3.

Signed-off-by: Vikas Gupta <vikasg@netlogicmicro.com>
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/netlogic/xlp/platform.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/netlogic/xlp/platform.c b/arch/mips/netlogic/xlp/platform.c
index 2a87bb5..8e9de10 100644
--- a/arch/mips/netlogic/xlp/platform.c
+++ b/arch/mips/netlogic/xlp/platform.c
@@ -218,7 +218,7 @@ static int xlp_find_pci_dev(void)
 						pres[0].flags	= IORESOURCE_MEM;
 						irt = (nlm_hal_read_32bit_reg(mmio, DEV_IRT_INFO) & 0xFFFF);
 						printk(KERN_WARNING "XXXX IRT = %#x mmio = %#llx\n", irt, mmio);
-						irq = irt;	/* TODO */
+						irq = xlp_irt_to_irq(irt);
 
 						pres[1].start = irq;
 						pres[1].end = irq;
-- 
1.8.4.93.g57e4c17

